### PR TITLE
Lock version per plugin

### PR DESCRIPTION
**Pull request by ramereth**
This uses versions as they are currently installed in prod. This should work around the issue we're having with chef runs on jenkins1. Tests will come in a later PR as its ... complicated.

### CODE REVIEW

**Comment by ramereth** [comment]
@osuosl-cookbooks/chefs please review

File: recipes/cookbook_uploader.rb [hunk 1]
```diff
@@ -64,7 +72,7 @@
 end
 
 # This is necessary in order to create Jenkins jobs with security enabled.
-node.run_state[:jenkins_private_key] = secrets['jenkins_private_key']
+node.run_state['jenkins_private_key'] = secrets['jenkins_private_key']
```
Comment (armiller) [inline recipes/cookbook_uploader.rb]: This will cause jenkins permissions to fail. It needs to be a symlink. I had to debug this before.

Comment (ramereth) [inline recipes/cookbook_uploader.rb]: I will add an exclusion line for foodcritic here then.

**Comment by ramereth** [comment]
@osuosl-cookbooks/chefs please look again

**Comment by armiller** [comment]
👍

**Comment by ramereth** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.7.23.  Have a nice day!

**Pull request merged**